refactor: migrate docs of individual services to documentations repo

francis-tanyx opened the pull request:
## PR Checklist
<!-- Please check if your PR fulfills the following requirements: -->

- [ ] Added label to the Pull Request for easier discoverability and search
- [ ] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)
- [ ] No Employer Confidential details are added

## What requirement is this design document for?

#**Issue #51**

## Anything the reviewer should know when reviewing this PR?

- This PR will close [issue #51](https://github.com/intel-retail/documentation/issues/51).
- Page heading and subheading were added for `pipeline-server.md` as it did not originally had any
- Page heading was added for `retail-data-visualization.md` as it did not originally had any

## If there are any other design Pull Requests or requirements, please link them here (i.e. intel-retail/automated-self-checkout )

https://github.com/intel-retail/automated-self-checkout/pull/677

francis-tanyx commented:
cleaned up commits in this new PR

ejlee3 reviewed (approved):
LGTM

NeethuES-intel reviewed (dismissed):
LGTM

ejlee3 reviewed (changes requested):
Can we grab the latest file from the repository - some things changed from a [PR](https://github.com/intel-retail/automated-self-checkout/pull/684) that was merged from the OSS hackathon in the common-service.md and retail-data-visualization.md was removed as well.

NeethuES-intel commented:
@francis-tanyx are the changes suggested by @ejlee3 incorporated in this PR. We need to merge this first & then the PR - https://github.com/intel-retail/automated-self-checkout/pull/704

francis-tanyx commented:
> @francis-tanyx are the changes suggested by @ejlee3 incorporated in this PR. We need to merge this first & then the PR - [intel-retail/automated-self-checkout#704](https://github.com/intel-retail/automated-self-checkout/pull/704)

Yes I confirm the changes are done, 
- removed the old retail-data-visualization.md 
- updated common-service.md

unsure if the contents in retail-data-visualization.md is migrated
